website/content/en/docs/upgrading-sdk-version/version-upgrade-guide.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Operator SDK Version upgrade guide
 linkTitle: v0.2.x to v0.17.x
-weight: 4
+weight: 1999982000
```

Comment:
Let's make this equivalent to the v0.17.0 migration guide since that's the most recent version it covers. Therefore:
```suggestion
weight: 999983000
```